docs/connect.html

```diff
@@ -47,7 +47,7 @@
 
 The remaining parameters are connector configuration files. You may include as many as you want, but all will execute within the same process (on different threads).
 
-Distributed mode handles automatic balancing of work, allows you to scale up (or down) dynamically, and offers fault tolerance both in the active tasks and for configuration and offset commit data. Execution is very similar to standalone mode:
+Distributed mode handles automatic work balancing work, allows you to scale up (or down) dynamically, and offers fault tolerance both in the active tasks and for configuration and offset commit data. Execution is very similar to standalone mode:
```

Comment:
Hmm, seems we've introduced a typo -- "work balancing work". What was wrong with the original version of this sentence?